### ISSUE TITLE
Easier Access to Hidden Apps

### ISSUE THREAD

**Issue by InsouciantQualms** [8bitPit/Niagara-Issues#2798]
### Is your feature request related to a problem?

I have the majority of my apps hidden.  The apps are categorized into three buckets:  either used all the time (in favorites), semi-frequently (still in the all apps list) or used less than weekly (in the hidden apps).

My issue is how many clicks it takes to get to the hidden apps to open one.  At least one long press to see Niagara Settings, then tap that, then Features, and then Hidden Apps.

### Describe the solution you'd like

My request is to make the Hidden Apps accessible (just like they are when adding a Popup/Category) at the bottom of the All Apps list (the diamond).  

If you look, there is an inconsistency already.  When adding a Popup, you do have the option to "Show Hidden Apps".  However, nearly the same UX in the all apps drawer does not have the option to show hidden apps.  This would make it a swipe and a click to make them visible, far easier.  Plus, the two different experiences (all apps and add popup) would now be the same.

TL/DR, this should simplify things rather than making them more complicated.  :^)

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned
